tests/test_models/test_mutables/test_gumbelchoiceroute.py

```diff
@@ -48,8 +51,10 @@ def test_forward_arch_param(self):
         new_gumbel_choice_route_cfg['with_arch_param'] = False
 
         new_gumbel_choice_route = MODELS.build(new_gumbel_choice_route_cfg)
+        mutator.prepare_from_supernet(new_gumbel_choice_route)
```

Comment:
Use user-defined instead  of `mutator`  to construct `arch_param`, as we hope the mutable could work(build and forward) independently in our design.